spec/declaration.dd

```diff
@@ -55,6 +55,12 @@ $(GNAME DeclaratorIdentifierList):
 $(GNAME DeclaratorIdentifier):$(LEGACY_LNAME2 VarDeclaratorIdentifier)
     $(GLINK_LEX Identifier)
     $(GLINK_LEX Identifier) $(GLINK2 template, TemplateParameters)$(OPT) $(D =) $(GLINK Initializer)
+    $(GLINK BitfieldDeclarator)
+
+$(GNAME BitfieldDeclarator):
+    $(D :) $(GLINK2 expression, AssignExpression)
+    $(GLINK_LEX Identifier) $(D :) $(GLINK2 expression, AssignExpression)
+    $(GLINK_LEX Identifier) $(D :) $(GLINK2 expression, AssignExpression) $(D =) $(GLINK Initializer)
```

Comment:
Update for @tim-dlang's observation (fixed):
```suggestion
    $(GLINK_LEX Identifier) $(D :) $(GLINK2 expression, ConditionalExpression)
    $(GLINK_LEX Identifier) $(D :) $(GLINK2 expression, ConditionalExpression) $(D =) $(GLINK Initializer)
```

Reply:
See https://github.com/dlang/dlang.org/pull/3190/files/ccdacb6866cd19d3617ec13d4792525e496c85b0#r884139913

Reply:
yes